### ISSUE TITLE
Relying on name rather than steamid

### ISSUE THREAD

**Issue by pspchucky** [Patcher56/PatchProtect#110]
https://github.com/Patcher56/PatchProtect/blob/18faeaf421e67708d09e77cdc925e5648b956f7a/lua/patchprotect/server/antispam.lua#L69-L71
```lua
local rep = string.Replace( sv_PProtect.Settings.Antispam[ "concommand" ], "<player>", name )
local cmd = string.Explode( " ", rep )
RunConsoleCommand( cmd[1], unpack( cmd, 2 ) )
```

Really bad idea, multiple players can have the same or similar names.
Earlier we ran into this problem where someone was changing their name to 3 letters of who they also wanted to match.
Example:
```
Xua
XuavTheGreat
```
The spammer, Xua, would intentially spam to execute the concommand which when executed, caused the admin mod to find 2 matching players and kill both. Things got worse when the spammer changed his name to the wildcard for everyone ( which is '*' in our admin mod )

TL;DR: Passing the player name to RunConsoleCommand is a bad idea as most admin mods will find multiple people with a similar name. Use SteamID instead or add <steamid> as an option.

**Comment by Domii894** [comment]
Hey pspchucky,
thank you for your bug report.
I just sent a commit to github. The problem is, that this is untested, because I am currently not able to check if it works or not (for some days actually). Would you be so kind and test it for me, so patcher and me can give it a go for workshop?

**Comment by pspchucky** [comment]
Confirm it works for Evolve, Existo, and ULX but not standard gmod-commands as they require the player name (which is :-1: :cry:)
````
] kick
Usage:  kick < name >

````

**Comment by Domii894** [comment]
Hmm, ok thanks!
I will let it at this state for some days and will have a look how to overcome this error.
